src/kr_comp.cpp

```diff
@@ -4,8 +4,12 @@ using namespace Rcpp;
 using std::string;
 // Obtain P,Q,R element from a mmrm fit, given theta.
 List get_pqr(List mmrm_fit, NumericVector theta) {
+  auto as_num_matrix_tmb = as_matrix<matrix<double>, NumericMatrix>;
```

Comment:
I guess we can't define these shortcuts globally right?

Reply:
if defined globally there will be conflicts; to be defined globally we do need the function name in header file and the function definition in a cpp file.

what about just using `as_matrix<matrix<double>>()` ? this is shorter and the other type can be deduced.
Further more, we could `typedef matrix<double> mxd;` annd then `as_matrix<mxd>` ?

Reply:
hm ok then maybe we keep as is